rage/src/bin/rage/cli.rs

```diff
@@ -152,5 +155,6 @@ pub(crate) struct AgeOptions {
     #[arg(short, long)]
     #[arg(value_name = fl!("output"))]
     #[arg(help = fl!("help-flag-output"))]
+    #[arg(value_hint = ValueHint::AnyPath)]
```

Comment:
Same question here as above about how this interacts with `-`.

By contrast to above, here if the output file exists it will be overwritten, so `AnyPath` (or `FilePath`) is correct.